clients/src/main/java/org/apache/kafka/clients/consumer/internals/StreamsRebalanceListener.java

```diff
@@ -22,7 +22,7 @@
 /**
  * Callbacks for handling Streams group rebalance events in Kafka Streams.
```

Comment:
maybe replace callback by listener in the comment as well?

Reply:
Done